envs/envs.go

```diff
@@ -229,9 +229,11 @@ func extractRelationshipsEnvs(env Environment) Envs {
 				values[fmt.Sprintf("%sSCHEME", prefix)] = endpoint["scheme"].(string)
 				values[fmt.Sprintf("%sNAME", prefix)] = endpoint["path"].(string)
 				values[fmt.Sprintf("%sDATABASE", prefix)] = endpoint["path"].(string)
+				values[fmt.Sprintf("%sDB", prefix)] = endpoint["path"].(string)
 				values[fmt.Sprintf("%sUSER", prefix)] = endpoint["username"].(string)
 				values[fmt.Sprintf("%sUSERNAME", prefix)] = endpoint["username"].(string)
 				values[fmt.Sprintf("%sPASSWORD", prefix)] = endpoint["password"].(string)
+				values[fmt.Sprintf("%sURL", prefix)] = fmt.Sprintf("%s://%s:%s@%s:%s/?authSource=%s", endpoint["scheme"].(string), endpoint["username"].(string), endpoint["password"].(string), endpoint["host"].(string), formatInt(endpoint["port"]), endpoint["path"].(string))
```

Comment:
Just a suggestion: Put each parameter on separate lines as this is a veeery long line :)

Reply:
That's true. I decided to follow what's been done for amqp, but that can be improved.